### PR TITLE
Migrate document to YAML with explicit chapter/paragraph/list numbering

### PR DESCRIPTION
**Pull request by Iipin**
`diffPDF` found no differences between the new document and the old one.
Nice potential follow-ups:
* There are some LaTeX constructs hardcoded (e.g. `\subsection`s), it'd be nice if we could express them in a better way,
* ~~Adjust the workflows,~~
* ~~Get rid of the pandoc,~~
* After the above: clean up the LaTeX template file.

### CODE REVIEW

File: statut.yaml [hunk 1]
```diff
@@ -0,0 +1,632 @@
+ chapters:
+  - id: "1"
+    title: "Postanowienia ogólne"
+    paragraphs:
+      - id: "1"
+        content:
+          - text: "\\textbf{Stowarzyszenie Krakowski Klub Chombo}, zwane dalej \\textbf{Stowarzyszeniem}, działa na podstawie ustawy Prawo o stowarzyszeniach (Dz. U. 1989 Nr 20 poz. 104 wraz z późniejszymi zmianami) oraz niniejszego Statutu."
```
Comment (m4tx) [inline statut.yaml:7]: I'm thinking, is it necessary to have distinction `text`/`list`? In YAML, these are already two distinct data structures, so maybe we can drop these tags and make the format less verbose?

**Review by m4tx — CHANGES_REQUESTED** [review]
Some downsides:
1. We can't easily generate docx files anymore
2. YAML format is a bit harder to read and write than the old Markdown files

We should re-enable CI (preferably using GH actions) before merging this.

File: statut.yaml (hunk 1 above)
Comment (Iipin) [reply statut.yaml:7]: yeah, we can – I automatically went for a tagged union so that we add another cases but no other cases came up. xd ptal

**Comment by Iipin** [comment]
> Some downsides:
> 
>     1. We can't easily generate docx files anymore
> 
>     2. YAML format is a bit harder to read and write than the old Markdown files
> 
> 
> We should re-enable CI (preferably using GH actions) before merging this.

1. docx files were added for mr. prawnik anyway so I figured we don't need them anymore
2. true

File: statut.yaml (hunk 1 above)
Comment (m4tx) [reply statut.yaml]: tbh now that I'm looking at this, it's hard to spot lists in this :eyes: At this point, I'm fine with either version.

**Comment by m4tx** [comment]
@Iipin what's the current status of this?

**Comment by Iipin** [comment]
@m4tx ready for rereview, PTAL

**Review by m4tx — APPROVED** [review]

**Pull request merged**